src/controllers/game.ts

```diff
@@ -3,6 +3,8 @@ import { ECharacterState, EShowOptions, EUserActions, EUserRace } from '../enums
 import { handleAttackEnemy, show as showFn } from './responses';
 import type { IAvailableCommands, IFightEntity, IFightTeam, IUserProfile } from '../types';
 import { createFight, getActiveFight, initProfile, leaveFight, saveLog, sendMessage } from '../communication';
+import type { AxiosResponse } from 'axios';
+import type { IDefaultResponse } from '../types';
```

Comment:
Try to integrate this import with already existing imports. On line 5, all elements are also imported from types

Reply:
Integrate import on line 7 to imports on 4 line